src/sortingtraining/Strategy.java

```diff
@@ -32,11 +40,26 @@ SortingStrategy resolve() {
             return new InsertionSort();
         }
     },
-    DEFAULT {
+    SHELL {
+        SortingStrategy resolve() {
+            return new ShellSort();
+        }
+    },
+    JDK_SORT {
         SortingStrategy resolve() {
             return new DefaultSort();
         }
     },
+    JDK_PARALLEL {
+        SortingStrategy resolve() {
+            return new DefaultSort();
```

Comment:
Shouldn't here be `DefaultParallelSort` ?

Reply:
Of course it should be parallel sort, thx for detecting this bug!